Main/cifar10_loader/Cifar_10_Loader.py

```diff
@@ -0,0 +1,41 @@
+import csv
+import pickle
+
+
+# Defining function for getting texts for every class - labels
+from termcolor import colored
+
+
```

Comment:
This file takes the original cifar10 pickle file and adds the missing labels to the dictionary.